pages/projects.js

```diff
@@ -51,7 +51,7 @@ export async function getServerSideProps (props) {
   try {
     const { loading, error, data: fetchProject } = await client.query({
       query: FETCH_ALL_PROJECTS,
-      fetchPolicy: 'network-only'
```

Comment:
Network-only sometimes uses the cache so it would show wrong reactions ( especially when user like/unlike the project and reload project list)

Reply:
But the default fetchPolicy is cache-first, It was a problem before that's why I added network-only instead. Are you sure this fixes the problem? https://www.apollographql.com/docs/react/data/queries/#setting-a-fetch-policy

Reply:
You know I had some problem with `network-only`, It sometimes was returning  the cache-data , I was confused as well, because based on Documentation it should not behave like that,
could you run this branch on your system?
then keep all changes that I made and try this page with two `no-cache` and `network-only`  fetchPolicy, to wee what I mean ( if with network-only was ok in your machine, change this, because on my system it sometimes fetched new response from server, and some time it was using cache and it made some messy result)
PS : I have other changes beside changing `fetchPolicy` please see them too

Reply:
it's weird no lies, but gotcha. it looks good to me